src/mjolnir/core/utility/alignment.h

```diff
@@ -0,0 +1,59 @@
+//! @file
+//! alignment.h
+//!
```

Comment:
Rename this file and the corresponding test file. Should be something with `pointer`. For example `pointer_utility`, `pointer_functions`, `pointer_tools`, etc.